src/Compiler/Facilities/Logger.fs

```diff
@@ -6,113 +6,15 @@ open System
 open System.Diagnostics
 open System.Diagnostics.Tracing
 
-module Activity =
-    
-    type ActivityFacade(activity : Activity option) =
```

Comment:
The `ActivityFacade` was brought in to seamlessly support a case where there are no listeners.
`activitySource.StartActivity(name)` returns `null` if there are no listeners.

Removing this causes NREs without listeners.

If such a facade is not desirable, an alternative would be to always hook up a dummy listener, or in other way support a scenario where no activity is returned.

Reply:
CC @T-Gro - we need to bring back support for no listeners in some way.

Reply:
Fixed now.